rj_gameplay/pass_costing/cost_heuristic.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import stp.rc as rc
+import stp.testing as testing
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+import math
+
+#max parameters for the various cost functions
+MAX_ROBOT_DISTANCE = 5
+MAX_CLOSEST_OPPONENT_DISTANCE = 6
+MIN_DOWNFIELD_DISTANCE = -2
+MAX_DOWNFIELD_DISTANCE = 3
+robot_distance_weight = 1
+closest_opponent_distance_weight = 1
+downfield_distance_weight = 0.5
+
+def cost_heuristic(world_state: rc.WorldState, target_robot: rc.Robot) -> float:
+    our_robots = world_state.our_robots
+    their_robots = world_state.their_robots
+    robot_with_ball = our_robots[0]
+    for robot in our_robots:
+        if robot.ball_sense_triggered:
+            robot_with_ball = robot
+            if robot_with_ball == target_robot:
+                return 99999
```

Comment:
Bit of a nit but if you want to return a number larger than all others use float('inf') instead